zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java

```diff
@@ -345,9 +370,13 @@ public Response createNote(String message) throws IOException {
       }
     }
     note.addNewParagraph(subject); // add one paragraph to the last
-    String noteName = request.getName();
-    if (noteName.isEmpty()) {
-      noteName = "Note " + note.getId();
+
+    //deal with possible empty note name
+    String noteName = "Note " + note.getId();
```

Comment:
Another option is to do in frontend that we can enforce user to provide a note name ? Personally, I prefer to do it in frontend to enforce user providing note name.